_data/locales/en.yml

```diff
@@ -55,10 +58,16 @@ ports_info: |
   Ports of openSUSE Leap to architectures other than the PC are
   maintained by separate community teams, with limited resources.
 jeos_images: JeOS
-jeos_info: For Arm JeOS images, please go to the <a href="https://en.opensuse.org/Portal:Arm#Release_supported_platforms">Arm wiki page</a>
+jeos_info: Minimal VM images. For Arm JeOS images, please go to the <a href="https://en.opensuse.org/Portal:Arm#Release_supported_platforms">Arm wiki page</a>
+minimal_vm_images: Minimal Virtual Machine
+minimal_vm_info: Similar to the SUSE Linux Enterprise Server Minimal-VM images. For Arm JeOS images, please go to the <a href="https://en.opensuse.org/Portal:Arm#Release_supported_platforms">Arm wiki page</a>
+minimal_aarch64: UEFI Arm 64-bit servers and boards (aarch64)
+minimal_x86_64: Intel or AMD 64-bit servers (x86_64)
```

Comment:
Those should still be called server_aarch64 and server_x86_64 imo

Reply:
Your wish is my command!